consul/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+exec 2>&1
+
+exec consul agent -dev
```

Comment:
I retract this comment having seen the nefarious scheme for dynamism in the run hook. 😄